packages/client/src/v2-events/features/events/Search/TabSearch.tsx

```diff
@@ -126,6 +126,24 @@ function countFilledFieldsForInputType(
   return value && value !== '' ? 1 : 0
 }
 
+function isFieldExcludedFromSearch(
+  field: FieldConfig,
+  currentEvent: EventConfig
+): boolean {
+  const fieldSearchConfig = currentEvent.advancedSearch
+    .flatMap((s) => s.fields)
+    .find((f) => f.fieldId === field.id)
+
+  if (
+    fieldSearchConfig?.fieldType === 'field' &&
+    fieldSearchConfig.excludeInSearchQuery
+  ) {
+    return true
+  }
+
+  return false
+}
+
```

Comment:
<img width="634" height="984" alt="image" src="https://github.com/user-attachments/assets/f5516edc-0016-498f-bac5-1b6cc2cf2bd3" />

With this, they will be counted as one field as filled